### PR TITLE
Update facia-end-to-end asset routes to match dev-build

### PR DESCRIPTION
**Pull request by OliverJAsh**

### CODE REVIEW

File: common/app/dev/DevAssetsController.scala [hunk 1]
```diff
@@ -30,10 +30,6 @@ object DevAssetsController extends Controller with ExecutionContexts {
     )
   }
 
-  def atFonts(path: String) = at(s"fonts/$path")
-  def atJavascripts(file: String) = at(s"javascripts/$file")
-  def atStylesheets(file: String) = at(s"stylesheets/$file")
-
```
Comment (OliverJAsh) [inline common/app/dev/DevAssetsController.scala]: These weren't used anywhere else.

**Comment by janua** [comment]
Was this the problem with `facia-end-to-end`?

**Comment by janua** [comment]
:+1:

**Pull request merged**